rllib/utils/tf_utils.py

```diff
@@ -247,6 +250,8 @@ class for.
         raise ImportError("Could not import tensorflow!")
 
     if framework == "tf2":
+        if not tf1.executing_eagerly():
```

Comment:
Add this here in case we are using this utility function outside of "regular" RLlib classes (such as RolloutWorker, which handles this itself).

Reply:
this likely is not gonna work in a util function.
``enable_eager_execution()`` needs to be called before any TF operations, and throws exceptions otherwise.
this usually needs to be enabled in ``main()``.

Reply:
True, but then it would fail on the next line anyways, which asserts that eager is enabled :) 
I added this, such that this util function can be more safely used in test cases, where we switch between eager and traced (and graph) execution to prove that something works for all frameworks.